deploy/deployctl/config.py

```diff
@@ -14,7 +14,8 @@ class Configuration:
     domain: str = None
     environment_tag: str = None
     authorized_networks: str = "0.0.0.0/0"
-    data_pipeline_output: str = None
+    data_pipeline_output: str = "gs://gnomad-browser-data-pipeline"
+    data_snapshot_bucket: str = "gs://gnomad-browser-elasticsearch-snapshots"
```

Comment:
Would be good to add the snapshot configuration to README here: https://github.com/broadinstitute/gnomad-browser/blob/007388ffe8568940f71720bbde20874f9c1bd9ce/data-pipeline/README.md?plain=1#L52